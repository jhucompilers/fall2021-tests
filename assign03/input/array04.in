PROGRAM array04;
VAR a : ARRAY 10 OF INTEGER;

BEGIN
  a[-1] := 42;
END.
