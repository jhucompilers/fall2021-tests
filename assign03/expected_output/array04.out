0,VAR,a,ARRAY 10 OF INTEGER
